### PR TITLE
Pre-release changes for 0.8.0-rc1

### LINKED ISSUE #3312: TODO: bump version of pyln etc before release

### PR DESCRIPTION
**Pull request by cdecker**
Closes #3312 

Changelog-None

### CODE REVIEW

**Comment by cdecker** [comment]
For some reason it didn't include the sendonion changelog entry. I'll go through the logs and cross-check them again, just for safety.

File: CHANGELOG.md [hunk 1]
```diff
@@ -514,6 +560,7 @@ There predate the BOLT specifications, and are only of vague historic interest:
 7. [0.5.2] - 2016-11-21: "Bitcoin Savings & Trust Daily Interest II"
 
 [Unreleased]: https://github.com/ElementsProject/lightning/compare/v0.7.3...HEAD
```
Comment (niftynei) [inline CHANGELOG.md]: update this diff too?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,22 +4,68 @@ All notable changes to this project will be documented in this file.
...
+ - quirks: Workaround LND's `reply_channel_range` issues instead of sending error. ([3264](https://github.com/ElementsProject/lightning/pull/3264))
+ - tools: A new command, `guesstoremote`, is added to the hsmtool. It is meant to be used to recover funds after an unilateral close of a channel with `option_static_remotekey` enabled. ([3292](https://github.com/ElementsProject/lightning/pull/3292))
+
 ### Changed
 
+:warning: The default network and the default location of the lightning home directory changed. Please make sure that the configuration, key file and database are moved into the network-specific subdirectory. :warning:
+
+ - config: Default network (new installs) is now bitcoin, not testnet. ([3268](https://github.com/ElementsProject/lightning/pull/3268))
+ - config: Lightning directory, plugins and files moved into `<network>/` subdir ([3268](https://github.com/ElementsProject/lightning/pull/3268))
+ - JSON API: The `fundchannel` command now tries to connect to the peer before funding the channel, no need to `connect` before `fundchannel` anymore ! ([3314](https://github.com/ElementsProject/lightning/pull/3314))
```
Comment (niftynei) [inline CHANGELOG.md]: this isn't technically correct -- you still need to call connect with the ip address and port except for nodes that are included in the seed nodes (i think, i'm not 100% sure) cc @darosior for clarification

Comment (cdecker) [reply CHANGELOG.md]: Added a short clarification that an address needs to be known.

**Comment by cdecker** [comment]
ACK https://github.com/ElementsProject/lightning/pull/3332/commits/44fd5193cb4d8f5af1b88a03f9e6ef9088d8b8bb

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,22 +4,68 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.8.0-rc1] - 2019-12-11: "Blockchain Good, Orange Coin Bad"
+
+This release named by @m-schmoock.
```
Comment (m-schmoock) [inline CHANGELOG.md]: @cdecker  If you want, you can note my full name: `Michael Schmoock @m-schmoock`

Also, though being used throughout the changelog, `This release named by XYZ` is not a correct sentence.
Suggestion: "This release **was** named by XYZ"

Comment (m-schmoock) [reply CHANGELOG.md]: fixes that: https://github.com/ElementsProject/lightning/pull/3334

**Pull request merged**